csharp/ApiDocs/docfx.json

```diff
@@ -11,7 +11,11 @@
       ],
       "dest": "api",
       "disableGitFeatures": false,
-      "disableDefaultFilter": false
+      "disableDefaultFilter": false,
+      "properties": {
+          "AllowUnsafeBlocks: "true"
```

Comment:
This line doesn't seem to right. You have three quotes here. Three is not an even number.